### DISCUSSION TITLE
Project by tag - does not work or am i the problem

### DISCUSSION THREAD

**Discussion by harryinet** [marcusolsson/obsidian-projects#801] [Help] [answered]
May bethis is no issue ...
I create a project based on "tag", not "folder". 
No note is listet but i have a lot notes with the selected tag like "project" or "priority".
Did i something or something not to prevent it to work?

**Comment by Acylation** [comment, marked as answer]
For a tag named in `daily`, you can build the project with the target tag field filled in the form `#daily`. Maybe double check the `#` sharp character and retry will solve this. If the problem persists, feel free follow up and you can post the project config and an example note for better reference.

**Comment by harryinet** [comment]
It works fine, found the discussion about tag tags and # ...
As often, the main problem is in front of the PC ;-)
